### PR TITLE
Merge `ElementList` components

### PR DESCRIPTION
**Pull request by vladimyr**
- deprecating editor's `ElementList` in favor of tce-core's `ElementList`
- moving editor drawer handling to `Container` component
- adding `list-add` slot inside `ElementList`'s footer to allow easy customization of `AddElement`

### CODE REVIEW

File: client/components/common/tce-core/ElementList.vue [hunk 1]
```diff
@@ -52,11 +61,11 @@ export default {
   },
   computed: {
     options() {
-      return {
+      return Object.assign(this.dragOptions, {
```
Comment (lukasaric) [inline client/components/common/tce-core/ElementList.vue]:
```suggestion
      options: vm => Object.assign(vm.dragOptions, {
      handle: '.drag-handle',
      scrollSpeed: 15,
      scrollSensitivity: 125
    }),
```

Also, you can write component `data` ☝️  as a one-liner to be consistent with the rest of the application :bowtie:

File: client/components/editor/structure/ContentContainers/Container.vue [hunk 2]
```diff
@@ -19,27 +19,41 @@
...
       :activity="container"
-      :types="types"
-      :layout="layout">
-      <template v-slot:list-item="{ item, dragged, setWidth }">
+      :supported-types="types"
+      :layout="layout"
+      class="element-list">
+      <template v-slot:list-item="{ element, isDragged, position }">
+        <inline-activator @click.native="showElementDrawer(position - 1)" />
         <content-element
```
Comment (lukasaric) [inline client/components/editor/structure/ContentContainers/Container.vue]:
```suggestion
	<content-element v-bind="{ isDragged, element, setWidth: false }" />
```
It's a preferable thing 🙃

File: client/components/editor/structure/ContentContainers/Container.vue [hunk 3]
```diff
@@ -89,4 +118,17 @@ export default {
   min-height: 2.25rem;
   margin-bottom: 0.5rem;
 }
+
+.element-list ::v-deep .contained-content {
+  margin: 0;
+}
+
+.element-list .sortable-drag {
+  margin: 10px 0;
```
Comment (lukasaric) [inline client/components/editor/structure/ContentContainers/Container.vue]: Replace `px` with `rem`.

File: client/components/editor/structure/ContentContainers/Container.vue [hunk 4]
```diff
@@ -64,22 +82,33 @@ export default {
...
+    reorder({ newPosition }) {
       const items = this.contentElements;
       const element = items[newPosition];
       const isFirstChild = newPosition === 0;
       const context = { items, newPosition, isFirstChild };
       this.reorderElements({ element, context });
     },
-    insert(element) {
+    showElementDrawer(position) {
+      this.insertPosition = position;
```
Comment (lukasaric) [inline client/components/editor/structure/ContentContainers/Container.vue:93]: Maybe `Object.assign` is cleaner? Also on method bellow 👇

**Comment by underscope** [comment]
@vladimyr  ?

**Pull request merged**